pkg/components/components.go

```diff
@@ -502,6 +502,7 @@ func GetOperatorCRD(namespace string) *extv1beta1.CustomResourceDefinition {
 				Singular:   "hyperconverged",
 				Kind:       "HyperConverged",
 				ShortNames: []string{"hco", "hcos"},
+				Categories: []string{"all"},
```

Comment:
Do you want to add "virt"?

Reply:
I do, but this is for all KubeVirt operators. So I want to open a discussion first.

Reply:
Also, since it's common to all the CRDs, we need to make sure about naming and if it's acceptable.

Reply:
This is just for HCO CRD (plus v2v CRDs that are only temporary here)